src/cap-description.adoc

```diff
@@ -119,11 +119,11 @@ The encoding in xref:cap_perms_encoding32[xrefstyle=short] is chosen to
 facilitate hardware implementations. Therefore, it can be worked out if the
 permissions are granted as follows:
 
-* <<c_perm>>: bit 0 is set
+* <<r_perm>>: bits 3 or 0 are set
```

Comment:
no - you've invalidated the encoding table above - you should have changed it on the CANDPERM page

Reply:
Oops you're right. Ok should be fixed now. It was easier to change here than in CAndPerm. I assume there's no reason to prefer one specific ordering for the RV64 bits over another?